test/create-article.ts

```diff
@@ -3,7 +3,7 @@ import { Iri, JsonLdObj } from 'jsonld/jsonld-spec';
 
 const isNotUndefined = (arg: unknown): boolean => arg !== undefined;
 
-export default (id?: Iri, name = `Article ${id}`): JsonLdObj => (
+export default (id?: Iri, name = id ? `Article ${id}` : 'Article'): JsonLdObj => (
```

Comment:
would expect a corresponding test that fails before the change and passes after the change?

Reply:
(assuming this is the correct behavior)

Reply:
I had considered adding a `create-article.test.ts`.

Reply:
I have added a failing test now and followed up with the fix.